pandas/io/gbq.py

```diff
@@ -30,9 +30,7 @@ def read_gbq(
     configuration: Optional[Dict[str, Any]] = None,
     credentials=None,
     use_bqstorage_api: Optional[bool] = None,
-    private_key=None,
```

Comment:
All tests passed, assuming it's safe to remove.

Reply:
The tests for gbq don’t run on development branches. unless you have the gbq tests setup locally?

Reply:
I did run all tests specified in `pandas/tests/io/test_gbq.py`, is there any other file i should explicitly run to verify this. Or any other means to verify this locally.